test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -479,6 +522,115 @@ void ProtoConverter::visit(AssignmentStatement const& _x)
 	m_output << "\n";
 }
 
+// Called at the time function call is being made
+template <class T>
+void ProtoConverter::visitFunctionInputParams(T const& _x, unsigned _numInputParams)
+{
+	// We reverse the order of function input visits since it helps keep this switch case concise.
+	switch (_numInputParams)
+	{
+		case 4:
```

Comment:
We usually do not indent the case labels.

Reply:
done